### DISCUSSION TITLE
How to restart app if im using GitHubUpdateManager?

### DISCUSSION THREAD

**Discussion by alejo9010** [Squirrel/Squirrel.Windows#1821] [General]
hello im triying to restart the app after it updates, but if i use application.restart(), it will start the old version, and i dont see any RestartApp method from GitHubUpdateManager, any solution ?

**Comment by owns** [comment]
Are you sure the update is being applied completely? I've only seen this when something went wrong with the update. You can set a breakpoint between `UpdateApp` and `restart`.
The `restart` method is pretty basic, so I'd recommend restarting "manually".  I personally like using `RestartAppWhenExited()` so I can ensure I shutdown appropriately (e.g. trigger a close event).
https://github.com/Squirrel/Squirrel.Windows/blob/87e3450d798bcb8d99ee829f89f63066c6919b8b/src/Squirrel/UpdateManager.cs#L205
